packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -121,6 +122,7 @@ export class BlockList extends Component {
 				style={ containerStyle }
 				onAccessibilityEscape={ clearSelectedBlock }
 			>
+				{ isRootList && <FloatingToolbar.Slot /> }
```

Comment:
Rendering slot only for RootList should eliminate render FloatingToolbar for each nested level

Reply:
Hmm, maybe we could move it somewhere else. i.e `packages/edit-post/src/components/layout/index.native.js`. Do you think it is possible?

Reply:
As discussed on call - moved it to layout component